frontend/src/modules/dashboard/views/HFNCMainView.tsx

```diff
@@ -57,6 +61,8 @@ const useStyles = makeStyles((theme: Theme) => ({
  */
 const HFNCMainView = (): JSX.Element => {
   const classes = useStyles();
+  const spo2AlarmLimits = useSelector(getSpO2AlarmLimitsCurrent, shallowEqual);
+  const hrAlarmLimits = useSelector(getHRAlarmLimitsCurrent, shallowEqual);
```

Comment:
Would it make sense to just provide the selector directly to `ValueInfo` the way you do for the value itself, or do we have to use `useSelector` here?

Reply:
That would mean changing the types in props, and since we currently use it in HFNCControl in MultiStepWizard too, I don't want to add another layer of complexity!

Reply:
Ok, we'll save this for a future refactor.